src/frontend-scripts/components/section-main/Profile.jsx

```diff
@@ -291,6 +292,11 @@ class ProfileWrapper extends React.Component {
 			if (wins + losses < 50) {
 				gamesUntilRainbow = 50 - wins - losses;
 			}
+
+			const { winsSeason = 0, lossesSeason = 0 } = user;

```

Comment:
I know this is just the same as the old format, and that's fair, but this is pretty redundant - we can just set the value directly, and only show the counter if the number is positive.